### ISSUE TITLE
ComboBox selection via typing only does not work as expected

### ISSUE THREAD

**Issue by brunnerh** [carbon-design-system/carbon-components-svelte#1222]
E.g. taking the [default example from the docs](https://carbon-components-svelte.onrender.com/components/ComboBox#default):

```svelte
<script>
  import { ComboBox } from "carbon-components-svelte";
</script>

<ComboBox
  titleText="Contact"
  placeholder="Select contact method"
  items={[
    { id: "0", text: "Slack" },
    { id: "1", text: "Email" },
    { id: "2", text: "Fax" },
  ]}
/>
```

If one types `Email` and presses Enter, the component will select `Slack` instead. Highlighting and selecting different items only works via arrow keys, which defeats the point of having a text-based input.

The [React implementation](https://react.carbondesignsystem.com/?path=/story/components-combobox--combobox-story) gets this somewhat right:
- When e.g. `Option 2` is typed, the corresponding option is highlighted
- When Enter is pressed the highlighted option is selected

Ideally it should highlight `Option 1` as soon as `Option` is typed, which does not happen. It also hijacks the Home and End keys, which I do not approve of, as those can be useful for editing the text in the field.

Tested on Windows 10 in Chrome and Firefox.

**Comment by patoi** [comment]
@brunnerh @metonym I think I could resolve this issue.

**Comment by metonym** [comment]
Fixed in [v0.67.5](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.67.5)
